packages/wonder-blocks-link/src/components/link.js

```diff
@@ -127,19 +127,16 @@ type CommonProps = {|
     onKeyUp?: (e: SyntheticKeyboardEvent<>) => mixed,
 |};
 
-export type SharedProps =
```

Comment:
Exporting a shared type that include conditional props makes it difficult to use in `LinkCore`.  Instead we only export the common props (like button.js) and then add any missing props to `LinkCore`'s props manually.